src/components/PhotosPicker.vue

```diff
@@ -20,51 +21,45 @@
  -
  -->
 <template>
-	<div class="photos-picker">
-		<div class="photos-picker__content">
-			<nav class="photos-picker__navigation" :class="{'photos-picker__navigation--placeholder': monthsList.length === 0}">
-				<ul>
-					<li v-for="month in monthsList"
-						:key="month"
-						class="photos-picker__navigation__month"
-						:class="{selected: targetMonth === month}">
-						<NcButton type="tertiary" :aria-label="t('photos', 'Jump to {date}', {date: dateMonthAndYear(month)})" @click="targetMonth = month">
-							{{ month | dateMonthAndYear }}
-						</NcButton>
-					</li>
-				</ul>
-			</nav>
-
-			<FilesListViewer class="photos-picker__file-list"
-				:class="{'photos-picker__file-list--placeholder': monthsList.length === 0}"
-				:file-ids-by-section="fileIdsByMonth"
-				:empty-message="t('photos', 'There are no photos or videos yet!')"
-				:sections="monthsList"
-				:loading="loadingFiles"
-				:base-height="100"
-				:section-header-height="50"
-				:scroll-to-section="targetMonth"
-				@need-content="getFiles"
-				@focusout.native="onFocusOut">
-				<template slot-scope="{file, height, isHeader, distance}">
-					<h3 v-if="isHeader"
-						:id="`photos-picker-section-header-${file.id}`"
-						:style="{ height: `${height}px`}"
-						class="section-header">
-						{{ file.id | dateMonthAndYear }}
-					</h3>
-
-					<File v-else
-						:file="files[file.id]"
-						:allow-selection="true"
-						:selected="selection[file.id] === true"
-						:distance="distance"
-						@select-toggled="onFileSelectToggle" />
+	<NcDialog content-classes="photos-picker"
+		:name="name"
+		:open="open"
+		out-transition
+		size="large"
+		@update:open="(open) => $emit('update:open', open)">
+		<!-- Navigation containing the months available -->
+		<template #navigation="{ isCollapsed }">
+			<!-- Mobile view -->
+			<NcSelect v-if="isCollapsed"
+				v-model="targetMonth"
+				:aria-label-combobox="t('photos', 'Jump to specific date in list')"
+				class="photos-picker__navigation__month-select"
+				:clearable="false"
+				:options="monthsList">
+				<template #selected-option="{ label }">
+					{{ dateMonthAndYear(label) }}
 				</template>
-			</FilesListViewer>
-		</div>
-
-		<div class="photos-picker__actions">
+				<template #option="{ label }">
+					{{ dateMonthAndYear(label) }}
+				</template>
+			</NcSelect>
+
+			<!-- Default view -->
+			<ul v-else>
```

Comment:
```suggestion
			<ul v-else aria-label="t('photos', 'Dates')">
```
For screen reader users similar to above